src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxEquivalenceTests.cs

```diff
@@ -437,5 +437,15 @@ public void TestUpdateInterpolatedString()
             VerifyEquivalent(tree1, tree2, topLevel: true);
             VerifyNotEquivalent(tree1, tree2, topLevel: false);
         }
+
+        [Fact(DisplayName = "https://github.com/dotnet/roslyn/issues/7380")]
+        public void TestExpressionBodiedMethod()
```

Comment:
What happens if there is a syntax error in the arrow clause?  Should we have a test for that?